### ISSUE TITLE
[Issue]: Installation always says it can't install facexlib, gfpgan, or realesrgan

### ISSUE THREAD

**Issue by NoppaiKohai** [vladmandic/automatic#271]
### Issue Description

![image](https://user-images.githubusercontent.com/7308004/233281168-bc7e024a-5147-4602-b6b8-78d3e129ef49.png)

The installer always says it hangs on that part. I have a working Automatic1111 already, so I have Python. I downloaded CUDA and it's working fine too. 

When I run the git clone and then run webui.bat it will throw errors on those three items. I can launch the webui.bat and eventually get in to the gradio where I can use it, but something is pretty wrong because I cannot use any LORA. 

If I use a LORA it says 

AttributeError: 'Options' object has no attribute 'lora_apply_to_outputs'  

![image](https://user-images.githubusercontent.com/7308004/233291057-69562cfa-9844-4ee4-89a5-122d80946565.png)
![image](https://user-images.githubusercontent.com/7308004/233291199-7babec8f-e589-460a-9adb-3434b8fe485c.png)

My system info page on Vlad if it helps 

![image](https://user-images.githubusercontent.com/7308004/233291412-523a7e2b-1109-4b8d-a949-dbd557350922.png)

### Version Platform Description

 Version: e6701f5c Wed Apr 19 14:12:19 2023 -0400

OS: Windows 10

Browser: Firefox

**Comment by vladmandic** [comment]
this is expected and fully documented, please search before creating new issues: #110
tl;dr, you're using unsupported version of python.

**Comment by scooller** [comment]
I have same issue but Python 3.10.11

**Comment by vladmandic** [comment]
> I have same issue but Python 3.10.11

I doubt its the same.
Update repo to latest version, run setup and report create new issue if necessary.

**Comment by scooller** [comment]
it the last version the problem is another extension, it https://github.com/opparco/stable-diffusion-webui-composable-lora I disabled it and it worked fine.
The bad thing is that I can't use it

**Comment by vladmandic** [comment]
Check if there is an issue already for thar extension and if not, create a new one and document the issue.

**Comment by scooller** [comment]
i found https://github.com/opparco/stable-diffusion-webui-composable-lora/issues/17
